main.go

```diff
@@ -97,14 +99,14 @@ func main() {
 			log.Info("Received signal:", sig)
 			if job == nil {
 				log.Warn("Info didn't get parsed from the job file, can't clean up. Probably don't need to.")
+				os.Exit(-1)
```

Comment:
I know this is only sorta related to your PR, but we should make sure that we've always created (if not written anything to) all the files listed in [transfer-output-files](https://github.com/cyverse-de/job-templates/blob/e5abbbe40a709b6ef762ce8ea61c9c56489a5bbb/condor_templates.go#L77), so that we can be sure the logs will always come back to the submit node at least. I think that in this case, they wouldn't -- and I'm not sure if that'd also prevent the output and stderr files from coming back properly.

Reply:
I'm not sure it makes a difference.

I just ran a test in dev where I canceled the job before parsing the job file, once without creating those files, and once creating empty files before cancelling, but the `logs` dir on the submit node had empty copies of those files in both cases. There were also 0-byte `script-error.log` and `script-output.log` files on the submit node in both cases 😕